pkg/controller/logcollector/logcollector_controller.go

```diff
@@ -366,14 +366,14 @@ func (r *ReconcileLogCollector) Reconcile(ctx context.Context, request reconcile
 		}
 	}
 
-	var pullSecrets, esSecrets []*corev1.Secret
+	pullSecrets, err := utils.GetNetworkingPullSecrets(installation, r.client)
+	if err != nil {
+		r.status.SetDegraded(operatorv1.ResourceReadError, "Error retrieving pull secrets", err, reqLogger)
+		return reconcile.Result{}, err
+	}
+	var esSecrets []*corev1.Secret
 	var esClusterConfig *relasticsearch.ClusterConfig
 	if variant == operatorv1.TigeraSecureEnterprise {
```

Comment:
We have two `if variant == operatorv1.TigeraSecureEnterprise` branches right after each other (with just the pull secrets between).

let's combine them to reduce code complexity.